inspirehep/modules/authors/static/js/authors/authors_submission_form.js

```diff
@@ -0,0 +1,74 @@
+define(function (require, exports, module) {
+
+  "use strict";
+
+  var $ = require("jquery");
+  var tpl_field_message = require('hgn!js/forms/templates/field_message');
+
+  function AuthorUpdateForm() {
+
+    this.$inspireForm = $('#authorUpdateForm');
+    this.init();
+  };
+
+  AuthorUpdateForm.prototype = {
+    init: function init() {
+      var that = this;
+      $('.add-element').click(function () {
+        setTimeout(that.attachHandlers.bind(that), 200);
+      });
+      this.attachHandlers();
+    },
+
+    attachHandlers: function attachHandlers() {
+      var that = this;
+      var $yearSelector = $("input[id$='start_year'], input[id$='end_year']").not("input[id*='__index__']");
+      
+      $yearSelector.change(function(event) {
+        that.onYearChange(event);
+      });
+    },
+
+    onYearChange: function onYearChange(event) {
+      var that = this;
+      var elem = event.target;
+      var elemValue = parseInt(elem.value, 10);
+      var elemId = elem.id;
+      var elemIdSplit = elemId.split('-');
+      var otherFieldId = '';
+      var otherFieldValue;
+      var compareValues = false;
+      
+      if ( elemIdSplit[elemIdSplit.length - 1] === 'start_year' ) {
+        otherFieldId = elemId.replace('start_year', 'end_year');
+        otherFieldValue = parseInt(document.getElementById(otherFieldId).value, 10);
+        compareValues = elemValue >= otherFieldValue;
+      } else {
+        otherFieldId = elemId.replace('end_year', 'start_year');
+        otherFieldValue = parseInt(document.getElementById(otherFieldId).value, 10);
+        compareValues = elemValue >= otherFieldValue;
+      }
+
+      var isNumber =  /^\d{4}$/.test(elemValue);
+      var isOtherNumber =  /^\d{4}$/.test(otherFieldValue);
+
+      if ( !isNumber || !isOtherNumber ) {
+        return;
+      }
+
+      if ( !compareValues ) {
```

Comment:
oh I see, the problem is that there was a mistake in the code so I had two times the same:

```
elemValue >= otherFieldValue;
```

I have now amended it so that is correct and the comparison is different depending if you are in `start_year` or in `end_year`